packages/client/src/i18n/messages/views/userSetup.ts

```diff
@@ -264,6 +280,26 @@ const messagesToDefine: IUserSetupMessages = {
     defaultMessage: 'Show next {pageSize} of {totalItems}',
     description: 'Label for show more link',
     id: 'user.profile.auditList.showMore'
+  },
+  assignedAuditAction: {
+    defaultMessage: 'Assigned',
+    description: 'Description for declaration assignment',
+    id: 'user.profile.auditList.assigned'
+  },
+  unAssignedAuditAction: {
+    defaultMessage: 'Unassigned',
+    description: 'Description for declaration not assigned to self',
+    id: 'user.profile.auditList.unAssigned'
+  },
+  correctedAuditAction: {
+    defaultMessage: 'Corrected',
+    description: 'Description for declaration corrected',
+    id: 'user.profile.auditList.corrected'
+  },
+  archivedAuditAction: {
+    defaultMessage: 'Archived',
+    description: 'Description for declaration archived',
+    id: 'user.profile.auditList.archived'
```

Comment:
@favour234 can you create a pull request to Farajaland with translations added for all new language items?

We've used https://www.deepl.com/translator for translating french language items

Reply:
alright..